src/unit/test_zipmap.c

```diff
@@ -0,0 +1,146 @@
+#include "../zipmap.c"
+#include "test_help.h"
+
+static void zipmapRepr(unsigned char *p) {
+    unsigned int l;
+
+    printf("{status %u}",*p++);
```

Comment:
It looks like you removed all the prints except this one?

Reply:
Oh sorry, I missed to remove!